docs/builders/index.mdx

```diff
@@ -36,8 +36,8 @@ Then, run [`packer init`](/docs/commands/init).
 packer {
   required_plugins {
     vmware = {
-      version = ">= 1.0.3"
-      source = "github.com/hashicorp/vmware"
+      version = ">= 1.0.7"
+      source = "github.com/hashicorp/packer-plugin-vmware"
```

Comment:
This is actually the Packer plugin source address used by the `packer init` command. You can get a little more info on source addresses [here](https://www.packer.io/plugins#source-addresses). But the tl;dr is that it follows the following convention HOSTNAME/GITHUBORGNAME/PLUGINTYPENAME. In this case the plugin type is vmware. 

Since Packer plugin repositories are all prefixed with `packer-plugin` Packer can infer the full URL correctly.

```suggestion
      source = "github.com/hashicorp/vmware"
```

Reply:
That seems like bad system - why would you hard code incorrect URLs and then have the binary correct then instead of just making it accept correct URLs?

Reply:
This is design pattern for Packer to denote the builder. 

I would recommend opening the concern on the `hashicorp/packer` repo.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.